example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.7.0'
```

Comment:
this is probably fine to update, but look out for warnings in the gradle logs that mention multiple kotlin versions being on the classpath, which may happen if this version doesn't match what we've declared in the other build.gradle for the library